invenio_app_ils/mail/messages.py

```diff
@@ -59,7 +60,8 @@ def __init__(
         except TemplateError:
             kwargs["html"] = kwargs["body"]
 
-        footer_tmpl = current_app.jinja_env.get_template(footer_template)
+        footer_template = current_app.config.get("ILS_GLOBAL_MAIL_TEMPLATES", {}).get("footer")
```

Comment:
Just pay attention that when you use `dict.get` without default value, you might get `None`, so you need to handle that case:
```
footer_template = current_app.config.get(ILS_GLOBAL_MAIL_TEMPLATES, {}).get("footer")
if footer_template:
    footer_tmpl = current_app.jinja_env.get_template(os.path.join(self.GLOBAL_MAIL_TEMPLATES_BASE_DIR, footer_template))
    ... handle the footer ...
```
Watch out that footer is used below but it might None.